android-components/components/feature/push/src/main/java/mozilla/components/feature/push/AutoPushFeature.kt

```diff
@@ -95,23 +86,35 @@ class AutoPushFeature(
     // The preference that stores new registration tokens.
     private val prefToken: String?
         get() = preferences(context).getString(PREF_TOKEN, null)
-    private var prefLastVerified: Long
-        get() = preferences(context).getLong(LAST_VERIFIED, 0)
-        set(value) = preferences(context).edit().putLong(LAST_VERIFIED, value).apply()
 
     private val coroutineScope = CoroutineScope(coroutineContext) + SupervisorJob() + exceptionHandler { onError(it) }
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal var connection: PushManagerInterface? = null
     /**
      * Starts the push feature and initialization work needed. Also starts the [PushService] to ensure new messages
      * come through.
      */
     override fun initialize() {
         // If we have a token, initialize the rust component on a different thread.
         coroutineScope.launch {
+            if (connection == null) {
+                val databasePath = File(context.filesDir, DB_NAME).canonicalPath
+                PushManager(
+                    PushConfiguration(
+                        serverHost = config.serverHost,
+                        httpProtocol = config.protocol.toRustHttpProtocol(),
+                        bridgeType = config.serviceType.toBridgeType(),
+                        senderId = config.senderId,
+                        databasePath = databasePath,
+                        verifyConnectionRateLimiter = null
```

Comment:
Could we add an comment to the `verifyConnectionRateLimiter` parameter informing readers that this will default to 24 hours at the Rust layer?